integrations/azure-devops/azure_devops/gitops/port_app_config.py

```diff
@@ -0,0 +1,18 @@
+from port_ocean.core.handlers.port_app_config.models import (
+    PortAppConfig,
+    ResourceConfig,
+)
+from .file_entity_processor import GitSelector
+from pydantic import Field
+from typing import List, Literal
+
+
+class GitResourceConfig(ResourceConfig):
+    kind: Literal["repository"]
+    selector: GitSelector
+
+
+class GitPortAppConfig(PortAppConfig):
+    spec_path: List[str] = Field(alias="specPath", default=["port.yml"])
```

Comment:
allow single value and set default to single
check if **/port.yml is supported

Reply:
Seems like the current implementation is the name of the file without any regex, I will try to find wheter that's importatnt or not.
For now, made it to be deafulted to single string and changed types accordingly